android-sdk/build.gradle

```diff
@@ -2,83 +2,87 @@ apply plugin: 'com.android.library'
 apply plugin: 'me.tatarka.retrolambda'
 
 android {
-  compileSdkVersion 25
-  buildToolsVersion "25.0.2"
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
 
-  // allow inter-project inclusion of this library's debug build
+    // allow inter-project inclusion of this library's debug build
     // doesn't play well with deployment scripts
-  // publishNonDefault true
+    // publishNonDefault true
 
-  defaultConfig {
-    minSdkVersion 16
-    targetSdkVersion 25
+    defaultConfig {
+        minSdkVersion 16
+        targetSdkVersion 25
 
-    // versionCode and versionName are used by the SDK to identify the app to Bridge server
-    versionCode 1
-    versionName "version 1.0.2, build 1"
-  }
+        // versionCode and versionName are used by the SDK to identify the app to Bridge server
+        versionCode 1
+        versionName "version 1.0.2, build 1"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
```

Comment:
Side note: we should enable minify and code obfuscation in the future, but not obviously something for this PR